_includes/tutorials/survey-responses/ksql-test/code/tutorial-steps/test/expected-outputs/survey_analysis.log

```diff
@@ -0,0 +1,6 @@
++-----------------------------------------------------------------------+---------------------------------------------------------------------------+
+|SURVEY_QUESTION                                                        |RESULTS                                                                    |
++-----------------------------------------------------------------------+---------------------------------------------------------------------------+
+|What is your Favorite Thanksgiving Dish?                               |{Cranberry Jelly=1, Turkey=3, Green Beans=1, Stuffing=3}                   |
+|What is your Favorite Thanksgiving Pie?                                |{Pumpkin Pie=3, Pecan Pie=1, Apple Pie=4}                                  |
+Query terminated
```

Comment:
>or, you may even be able to `diff` the entire expected output (`survey_analysis.log`) with the actual output.

not sure if the results are in the same reliable order, but if so then a straight `diff` in the Makefile would work for the test condition